svgs/fix_price.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none"><path stroke="#000" stroke-linejoin="round" stroke-width="12" d="M80.662 170H41.195V22h44.4l.007 35.12h-4.86l-.08 23.094h4.934v35.52h-4.934zm70.241-89.79h-50.788v35.528h50.79zm1.903-58.177h-52.65v35.055h52.65z"/></svg>
\ No newline at end of file
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><path opacity=".9" d="M107 22v34m0 30v34m44-70V28a6 6 0 0 0-6-6H49a6 6 0 0 0-6 6v136a6 6 0 0 0 6 6h28a6 6 0 0 0 6-6v-38a6 6 0 0 1 6-6h48a6 6 0 0 0 6-6V92a6 6 0 0 0-6-6H89a6 6 0 0 1-6-6V62a6 6 0 0 1 6-6h56a6 6 0 0 0 6-6Z" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
Same reason here. Looks too generic without the key aspects of the original icon.